pom.xml

```diff
@@ -37,7 +37,7 @@
   </scm>
 
   <properties>
-    <java.version>1.6</java.version>
+    <minimal.java.version>6</minimal.java.version>
```

Comment:
Did this property actually require a rename?

Reply:
It is not required, just clarify the variable meaning. In fact, it may still not be quite clear - and I'd rename it to `target.java.version` (not to be confused with JDK version required for compilation).

Reply:
I agree, `target` is much more clear.

Reply:
To assist on a decision towards https://github.com/java-native/jssc/pull/81#discussion_r603325846, we may decide to do a preferred value which represents the logical minimum.  I'm not sure how badly that would conflate the XML.